docs/index.html

```diff
@@ -12,6 +12,12 @@
 	<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/4.1.0/css/fabric.min.css">
 
 	<style>
+		@media not all and (pointer:coarse) {
+			*:not([tabindex='-1']):focus {
+				outline: 2px solid #4D90FE !important;
+			}
+		}
```

Comment:
Why not stick with browser outline??

Reply:
I know that we chosen this color because of the feedbacks we had from micro:bit, but I feel it is the occasion to keep the same outline for all targets and our own website.
Would you prefer the native outline anyway?

Reply:
Yes native for the website.